packages/common/src/store/pages/profile/reducer.ts

```diff
@@ -346,8 +346,8 @@ const reducer = (
   const { currentUser, entries } = state
 
   const profileHandle =
-    'handle' in action
-      ? (action.handle?.toLowerCase() ?? currentUser)
+    'handle' in action && typeof action.handle === 'string'
```

Comment:
Fixes unrelated bug when a `track` action comes through the reducer w/ a `handle` prop that's an object not a string

Reply:
this is such a footgun